### PR TITLE
Language configuration

### LINKED ISSUE #167: Change captcha language
How to change captcha language ?
thx

### PR DESCRIPTION
**Pull request by reduardo7**
Fixes  #167

### CODE REVIEW

**Comment by TheSharpieOne** [comment]
Very interesting. This trigged recaptcha to fetch the language specific version of the script. (Previously, people have been trying to swap the scripts) I am not sure if that is a new capability or not.
Looking at how you accomplished this, you just set hl in the widget's internal recaptcha config. Doing this from the start (passing `hl` along with the config to `render`) appears to have the same affect... meaning there is an undocumented feature to set the language baked right into reCaptcha (not too sure if that is new or not).
The method you are using is very fragile though, it depends on internal script variables and those are not guaranteed to stay the same version to version. In fact, b/c the script is minified and mangled, I would be will to bet that this would break when there is an update to reCaptcha.
Basically, remove all of the hackiness which dives into `window.___grecaptcha_cfg` and it will still work but without being able to change an existing widget's language. You could watch for the lang to change and remove the existing widget and create a new one to have the ability to change the language of an 'existing widget' without going into the internal workings o recaptcha itself.

File: src/service.js [hunk 1]
```diff
@@ -157,14 +167,25 @@
...
 
                     // Let everyone know this widget has been reset.
                     $rootScope.$broadcast('reCaptchaReset', widgetId);
                 },
 
                 /**
+                 * Set reCaptcha language
+                 */
+                setLang: function (widgetId, lang) {
+                    var cli = window.___grecaptcha_cfg.clients[widgetId];
```
Comment (TheSharpieOne) [inline src/service.js]: things that start with `_` and especially `__` and even more so `___` are internal methods and should not be relied upon by third parties are they can change or disappear. `window.grecaptcha` is the official api.
Furthermore, variable names like `W` and `wk` should not be replied upon because those are generated by the build process and will change build to build, version to version.

**Comment by iambrosi** [comment]
@reduardo7 First, I want to thank you for the PR. I subscribe to what @TheSharpieOne mentioned, and I would like to add that all your changes must be performed on the files in the `src` dir. The files located in the `release` dir are generated during the release process that. For this same reason, you must not commit the files in the `release` dir.

Lastly, it would be awesome to have some tests included in the PR for the new feature.

**Comment by reduardo7** [comment]
@iambrosi and @TheSharpieOne : I have made the suggested changes, but I notice that the automatic integration test is failing for something that does not seem to be related to my work.
If you notice anything else that needs adjusting, do not hesitate to contact me.

**Comment by TheSharpieOne** [comment]
The issue is with the CI, travis.yml is set to use the latest stable version of node. That combined with out-dated devDep makes it not work.
I put up #171 to address that. If you want, once that gets merged, rebase this PR to include it so that travis CI will pass.

Also, I did test the merge result of this PR locally with the updated devDeps; everything passed.

**Comment by TheSharpieOne** [comment]
@iambrosi There should be in option to rebase this via the github buttons below.

**Comment by iambrosi** [comment]
Thanks @reduardo7 !

**Comment by reduardo7** [comment]
@iambrosi I agree with https://github.com/VividCortex/angular-recaptcha/pull/170#issuecomment-263983842, but now the `release` dir don't contain the minified code. Should I make this missing commit? Or you should do it?
